sentry/src/main/java/io/sentry/HubAdapter.java

```diff
@@ -39,7 +39,7 @@ public SentryId captureEnvelope(SentryEnvelope envelope, @Nullable Object hint)
 
   @Override
   public SentryId captureException(Throwable throwable, @Nullable Object hint) {
-    return Sentry.captureException(throwable, hint);
+    return Sentry.getCurrentHub().captureException(throwable, hint);
```

Comment:
I believe we don't need to change this one, right?

Reply:
You're right. thanks!